docs/commands/json.toggle.md

```diff
@@ -1,20 +1,74 @@
-Toggle a boolean value stored at `path`.
+Toggle a Boolean value stored at `path`
 
-return
+[Examples](#examples)
 
-@array-reply of @integer-reply - for each path, the new value (0-false or 1-true), or @nil-reply element for JSON values matching the path which are not boolean.
+## Required arguments
 
-@examples
+<details open><summary><code>key</code></summary> 
 
-```
-redis> JSON.SET doc $ '{"bool": true}'
+is key to parse.
```

Comment:
Key is not being parsed
```suggestion
is key to modify.
```

Also relevant for other occurrences.
For read-commands, such as `JSON.TYPE` or `JSON.ARRLEN`, etc.
which do not modify the key, we can use
```
is key to read.
```